_health-care/_js/_form.js

```diff
@@ -18,17 +18,60 @@ var HealthApp = (function() {
     var dateString = "";
 
     if (date.getMonth() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += (date.getMonth() + 1) + "/";
 
     if (date.getDate() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += date.getDate() + "/";
 
     return dateString + date.getFullYear();
   }
+  function extractMonetaryValueTransform(formData, arg) {
+    return Number(extractTransform(formData, arg)).toFixed(2).toString();
+  }
+  function validatePresence(input) {
+    var valid = true;
+
+    if (input.value === '') {
+      valid = false;
+    }
+
+    return valid;
+  }
+  function validateMonetaryValue(input) {
+    var valid = true;
+    var amount = input.value;
+
+    if (/[$a-zA-Z,]+/.test(amount)) {
```

Comment:
I'll check on the server-side to see if they do the same validation. Really the validation rules should be based on the Java server code, not the PDF form....so maybe leave the 99999999.99 check out for now.